common/src/main/java/com/wynntils/models/items/annotators/game/UnknownGearAnnotator.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.annotators.game;
+
+import com.wynntils.core.components.Models;
+import com.wynntils.handlers.item.ItemAnnotation;
+import com.wynntils.handlers.item.ItemAnnotator;
+import com.wynntils.models.gear.type.GearTier;
+import com.wynntils.models.gear.type.GearType;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.world.item.ItemStack;
+
+public final class UnknownGearAnnotator implements ItemAnnotator {
+    private static final Pattern UNKNOWN_GEAR_PATTERN = Pattern.compile("^§[5abcdef](.*)$");
```

Comment:
I hope this is strict enough. Probably is knowing all the checks we do before this annotator

Reply:
That is the reason this is the antepenultimate annotator we call...